src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64SimpleDotNet.apkdesc

```diff
@@ -5,28 +5,28 @@
       "Size": 3036
     },
     "classes.dex": {
-      "Size": 377764
+      "Size": 19476
```

Comment:
wait, what?  Why would enabling marshal methods impact *`classes.dex`*?!  I find this deeply concerning.  What types are *missing* due to this change, and *why*?